packages/agoric-cli/src/main.js

```diff
@@ -209,19 +209,6 @@ const main = async (progname, rawArgs, powers) => {
 
   baseCmd('follow <path-spec...>')
     .description('follow an Agoric Casting leader')
-    .option(
-      '--proof <strict | optimistic | none>',
-      'set proof mode',
-      value => {
-        assert(
-          ['strict', 'optimistic', 'none'].includes(value),
-          X`--proof must be one of 'strict', 'optimistic', or 'none'`,
-          TypeError,
-        );
-        return value;
-      },
-      'optimistic',
-    )
     .option(
```

Comment:
Please don't make an unnecessary breaking change here.  `--proof=none` is still legitimate.

```suggestion
    .option(
      '--proof <strict | optimistic | none>',
      'set proof mode',
      value => {
        assert.equal(
          value,
          'none',
          X`--proof can only be 'none'`,
          TypeError,
        );
        return value;
      },
      'none',
    )    
    .option(
```

Reply:
fair. I'll include something about the option being deprecated since it doesn't do anything.

Reply:
I edited the above to suggest something better than before.  Please take it into account.